src/sentry/tasks/post_process.py

```diff
@@ -702,8 +702,8 @@ def process_snoozes(job: PostProcessJob) -> None:
                 "user_count": snooze.user_count,
                 "user_window": snooze.user_window,
             }
-            add_group_to_inbox(group, GroupInboxReason.UNIGNORED, snooze_details)
-            record_group_history(group, GroupHistoryStatus.UNIGNORED)
+            add_group_to_inbox(group, GroupInboxReason.ONGOING, snooze_details)
+            record_group_history(group, GroupHistoryStatus.ONGOING)
```

Comment:
Not sure if this is something we should put behind the flag -- it'll make the migration harder since we'll keep writing rows with `GroupInboxReason.ONGOING`.

Reply:
we would need to support ONGOING in the frontend, we could swap between ongoing and unignored in the frontend but i'm not sure if that gets confusing

Reply:
Hm @scttcper would it work if I put this behind a flag so we don't write the new ONGOING status until the feature is on?

I believe the frontend would need to handle the new status as well as something like `issue_states_flag_enabled ? ONGOING : UNIGNORED`?

I can prep a migration for GroupInbox from UNIGNORED to ONGOING, and we can run it for organizations as we enable the feature.

Reply:
yeah that makes sense

Reply:
Any reason why we have to keep `GroupHistoryStatus.UNIGNORED` around? With this change, it looks like we won't be writing to `GroupHistory` with `GroupHistoryStatus.UNIGNORED`?

Reply:
Yep, we can remove it after the feature is enabled! Put the changes behind a flag for now. @scttcper I'll file a ticket to handle this on the FE